app/assets/stylesheets/components/lesson/lesson_content.scss

```diff
@@ -1,5 +1,9 @@
 .lesson-content {
 
+  section[id] {
```

Comment:
Add this to lesson-content.css file after https://github.com/TheOdinProject/theodinproject/pull/3137 is merged